Explainer/GPTExplainer.py

```diff
@@ -0,0 +1,74 @@
+
+from pptx import Presentation
+import os
+import openai
+import json
+import asyncio
+
+#api_key = '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' for excellentim
+API_KEY= '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' #excellentim tal
+
+# set the behaviour of the system
+messages = [
+    {"role": "system",
+     "content": "You should explain the slide content a user gives you, You should write it in a way"
+                "the age group provided can understand, when explaining the slide,"
+                "take into consideraton the extra information given to you, in order to explain"
+                "the slide even better."}
+]
+slide_explanation=[]
+openai.api_key = API_KEY
+
+
+'''
+gets response from chatgpt
```

Comment:
Write documentation in a format, this is an example of one of the acceptable formats
```python
def count_hi_v4(word: str) -> int:
"""
The function counts how many times the string "hi" appears in a given string.
@param word: The given string.
@return: The amount of times the word "hi" appeared.
"""
return word.count("hi")
```